File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.cs
public class Solution {
    public bool JudgeSquareSum(int c) {
        for(long num1 = 0; num1 * num1 <= c; num1++){
            var num2sq = c - num1*num1;
            // if this number is a perfect square, then we have a match
            var num2root = Math.Sqrt(num2sq);
            if((int)num2root == num2root) return true;
        }
        return false;
    }
}

File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.py
class Solution:
    def judgeSquareSum(self, c: int) -> bool:
        num1 = 0
        while num1*num1 <= c:
            num2sq = c - num1*num1
            # if this number is a perfect square, then we have a match
            num2root = sqrt(num2sq)
            if int(num2root) == num2root: return True
            num1 += 1
        return False